src/main/java/org/example/utils/Config.java

```diff
@@ -0,0 +1,32 @@
+package org.example.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.Properties;
+
+public class Config {
+    static String pathName = "src/test/resources/config.properties";
+
+    public static Properties getPropertyObject(String pathName) {
+        Properties prop = new Properties();
+        try {
```

Comment:
use try with resources:
`try (FileInputStream fp = new FileInputStream(path)){
            prop.load(fp);
        } catch (IOException e) {
            System.out.println("errors in the properties");
        }`